galsim/wcs.py

```diff
@@ -1038,6 +1088,29 @@ def x0(self): return self.origin.x
     @property
     def y0(self): return self.origin.y
 
+    def radecToImage(self, ra, dec, units, color=None):
```

Comment:
these versions avoid the order-of-arguments issue because the order is in the method name

Reply:
at least part of it, `radecToxy` would be even more explicit

Reply:
Sure, I'm fine with changing these names to include the order of the return values.